### DISCUSSION TITLE
Provide an ability to remove Extra or Excess Pools from the Chia Dashboard

### DISCUSSION THREAD

**Discussion by vdx660** [Chia-Network/chia-blockchain#8004] [Ideas]
I have a lot of extra Pools in the Dashboard that I don't need. Currently there's no ability to remove them.

**Comment by pailgrape** [comment]
yeah, and also there is no way to remove any plot directory in the gui, so add that too.

**Comment by keliew** [comment]
It's registered on the blockchain.

I guess one way is to 'remove' it from `config.yaml`.
